demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -80,16 +80,17 @@ def is_existing_image(self):
 
         if data_dictionary.get('image'):
             is_image_in_yml = True
-
         if not re.match(INTEGRATION_REGEX, self.file_path, re.IGNORECASE):
             package_path = os.path.dirname(self.file_path)
-            if is_image_in_yml:
-                print_error("You have added an image in the yml "
-                            "file, please update the package {}".format(package_path))
-                return False
             image_path = glob.glob(package_path + '/*.png')
             if image_path:
                 is_image_in_package = True
+        if is_image_in_package and is_image_in_yml:
```

Comment:
i think the condition should be that it is exactly in one, not in both

Reply:
I'm returning error if it's in both and if it's in none of the two, if it's in one- no exception will be raised, is that what you mean?

Reply:
yea, missed second condition
looks good